source/installing.rst

```diff
@@ -0,0 +1,233 @@
+
+The *Minor Release repository* includes a particular minor release of the database and all of the packages that were tested and verified to work with that minor release (e.g. ``pdps 8.0.19`` and ``pdpxc-8.0.19``). You may choose to install |pdmysql| from the Minor Release repository if you have decided to standardize on a particular release which has passed rigorous testing procedures and which has been verified to work with your applications. This allows you to deploy to a new host and ensure that you'll be using the same version of all the Distribution packages, even if newer releases exist in other repositories.
+
+The disadvantage of using a Minor Release repository is that you are locked in this particular release. When potentially critical fixes are released in a later minor version of the database, you will not be prompted for an upgrade by the package manager of your operating system. You would need to change the configured repository in order to install the upgrade.
+
+Prerequisites
+================================================================================
+
+Install |percona-release| if you haven't done it already. Otherwise, update it to the latest version. Please refer to `Percona repositories documentation <https://www.percona.com/doc/percona-repo-config/percona-release.html>`_ for installation or update instructions.
+
+Enable Percona repository       
+================================================================================
+
+We recommend to use the ``setup`` subcommand of |percona-release| to enable the desired repository.
+ 
+* For |PS|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdps-8.0
+
+* For |PXC|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdpxc-8.0
+
+.. hint::
+
+   To enable the minor version repository, use the following comand:
+
+   .. code-block:: bash
+   
+      $ #For PS-based deployment
+      $ sudo percona-release setup pdps-8.0.19
+      $ #For PXC-based deployment
+      $ sudo percona-relase setup pdpxc-8.0.19
+
+Make sure to run |percona-release| as root or via |sudo|. For the
+sake of convenience, all commands that require elevated privileges start with
+|sudo| in the following procedures.
+
+Install |pdmysql|
+================================================================================
+
+Install the selected deployment of |pdmysql| using the commands of the package manager of your operating system.
+
+.. important::
+
+   Make sure to run all commands as root or via |sudo|.
+
+Install |PS|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo apt-get install percona-server
```

Comment:
I think also we need to install client
percona-server-client
But not sure, maybe percona-server-client depends on percona-server.

Reply:
I guess it does https://repo.percona.com/pdps-8.0.19/apt/pool/testing/p/percona-server/

Reply:
Also we haven't percona-server package, only percona-server-server for server

Reply:
percona-server-client will be installed as the dependency for percona-server-server

Reply:
Will the above listed packages have to be installed explicitly?